tools/cdk-release/test/changelog.test.ts

```diff
@@ -68,6 +184,6 @@ function buildCommit(commit: PartialCommit): ConventionalCommit {
   };
 }
 
-async function invokeChangelogFrom1_23_0to1_24_0(args: ChangelogOptions, commits: ConventionalCommit[]): Promise<string> {
-  return (await changelog(args, '1.23.0', '1.24.0', commits)).contents;
+async function invokeChangelogFrom1_23_0to1_24_0(changelogArgs: ChangelogOptions, commits: ConventionalCommit[]): Promise<string | undefined> {
```

Comment:
Why this type change? `changelog()` still returns `Promise<string>`...?

Reply:
Holdout from an earlier attempt. Reverted.